media-sound/mumble/mumble-9999.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-inherit desktop multilib-build qmake-utils xdg
+inherit cmake xdg-utils
```

Comment:
Why did you replace `xdg.eclass` with `xdg-utils.eclass`?

Reply:
I don't really remember, but I think it was because I needed some functions that were not in the xdg class.
After the update I didn't need the xdg eclass

Reply:
`xdg.eclass` inhrits `xdg-utils.eclass` so I suppose you did it for some other reason. Anyway, please prefer `xdg.eclass`.